rbac/common/role/relationship_task.py

```diff
@@ -56,3 +56,13 @@ def related_type(self):
     def relationship_type(self):
         """The related type from AddressSpace implemented by this class"""
         return addresser.RelationshipType.MEMBER
+
+    @property
+    def _state_object_name(self):
+        """Task relationship state object name ends with Relationship (TaskRelationship)"""
+        return self._name_camel + "Relationship"
+
+    @property
+    def _state_container_list_name(self):
```

Comment:
and here.   I feel like this is a good use for base classing.  Why are you not adding _state_object_name and _state_container_list_name to the base class BaseRelationship?  Then there would not be as many repeats